Samples/V1.0Samples/LocalMediaSamples/AudioVideoPlaybackBot/README.md

```diff
@@ -12,63 +12,268 @@ This section walks you through the process of deploying and testing the sample b
+    host-header: localhost
+```
+
+Ngrok Pro Version:
+
+```json
+tunnels:
+  signaling:
+    addr: "https://localhost:9441"
+    proto: http
+    subdomain: <Your-Ngrok-Domain>
+    host-header: localhost
+```
+
+5.Now, setup a TCP tunnel to forward the traffic to localhost. Write the following lines into your ngrok.yml:
+
+Ngrok Free Account
+
+```json
+  media:
+    addr: 8445
+    proto: tcp
+```
+
+Ngrok Pro Account
+
+```json
+  media:
+    addr: 8445
+    proto: tcp
+    remote_addr: <Your-Ngrok-TCP-Address>
+```
+
+6.Your ngrok.yml now should look something like this:
+
+Ngrok Free Version
+
+```json
+authtoken: 43ksdfjlsielsdf_siwoeirwXXXX
+tunnels:
+  signaling:
+    addr: "https://localhost:9441"
+    proto: http
+    host-header: localhost
+  media:
+    addr: 8445
+    proto: tcp
+```
+
+Ngrok Pro Version
+
+```json
+authtoken: 43ksdfjlsielsdf_siwoeirwXXXX
+tunnels:
+  signaling:
+    addr: "https://localhost:9441"
+    proto: http
+    subdomain: contoso
+    host-header: localhost
+  media:
+    addr: 8445
+    proto: tcp
+    remote_addr: "1.tcp.ngrok.io:12332"
+```
+
+7.Now that the ngrok configuration is ready, launch it:
+
+`ngrok.exe start -all -config <Path to your ngrok.yml>`
+
+This starts ngrok and defines the public URLs which provide the tunnels to your localhost. The output looks like the following:
+
+Ngrok Free Account
+
+```cmd
+Forwarding  tcp://0.tcp.ngrok.io:15448 -> localhost:8445
+Forwarding  http://32f6055e17b3.ngrok.io -> https://localhost:9441
+Forwarding  https://32f6055e17b3.ngrok.io -> https://localhost:9441
+```
+
+Here, 9441 is the signaling port, 8445 is the application-hosted port, 32f6055e17b3 is a randomly generated ngrok domain name (note that it will change every time you restart ngrok), 0.tcp.ngrok.io:15448 is a randomly generated remote TCP address (note that it will change every time you restart ngrok).
+
+Ngrok Pro Account
+
+```cmd
+Forwarding  tcp://1.tcp.ngrok.io:12332 -> localhost:8445
+Forwarding  http://contoso.ngrok.io -> localhost:9441
+Forwarding  https://contoso.ngrok.io -> localhost:9441
+```
+
+Here, 9441 is the signaling port, 8445 is the application-hosted port, and 12332 is the remote media port exposed by ngrok. Of course, these port numbers are just examples and you can use any available port.
+
+#### Setting up domain and SSL certificates
+
+>Note: This section is only required for application-hosted media bots and can be skipped if you don't host media yourself.
+
+Application-hosted media uses certificates and TCP tunnels. The following steps are required:
+
+1. Create and register a domain name with one of the popular domain registrars. For our example here, we'll refer to it as: `contoso.com`
+
+2. A SSL certificate is required for your domain's based URLs. To make it easy, use a SSL certificate issued to a wildcard domain. In this case, it would be `*.contoso.com`. This SSL certificate is validated by the media SDK, so it should match your bot's public URL.
```

Comment:
sometimes wild card cert is hard to get for some customers. will be good to also call out specific URL option here.